### ISSUE TITLE
Server Metrics in Axiom

### ISSUE THREAD

**Issue by nang-dev** [trypear/pearai-server-issues-public#17]
Which metrics do we need in Axiom?

we have 3 categories
Free-trial (server)
Subscribed (server)
Own API-Key 

Let's get a graph of:
- Total Downloads
- Total requests (this should have 3 lines, different colors for each of the categories)
- Total Daily Active Users ^ -> # of unique users per day

Let's get a table of:

number of Requests for each category, then
number Requests for each model

- % of users that are in each category

How many users who used the app 1 week ago are still using it today?

- Are these users the same? Are they different?
- Who are our power users (using it 4 times a week)

-  todo: how many tokens does the average request use (input + output)?

Dot plot:

For all the top 100 users, lets make this visualization:

rows:
users (sorted descending by total # of requests)

cols (past 3 weeks):
each day: a color, green if they sent a request that day, grey if they didnt.

**Comment by Fryingpannn** [comment]
Basically some questions we need answers for:

- How many users who used the app 1 week ago are still using it today?
- How many people are converting from free trial to paid?
- Average chat credits spent from free trials
- Average chat credits spent from bought users
- Also cost per user in descending order from highest to lowest
- Average number of requests sent per user per month

**Comment by cqdev-co** [comment]
- [x] Total Downloads (can't get unique downloads without axiom log update for backend/frontend)
- [x] Total requests (this should have 3 lines, different colors for each of the categories)
- [x] Total Daily Active Users ^ -> # of unique users per day

Total Daily Active Users can be filtered based on the Axiom parameters. Ex: In the past 5 min there have been 6 users, 15 there have been 30 users, and etc. 

Table
- [x] # of Requests for each category
- [x] # Requests for each Model
- [ ] % of users that are in each category? (Not sure if that is possible but will try)

Basically some questions we need answers for:
- [ ]  How many users who used the app 1 week ago are still using it today?
- [ ] How many people are converting from free trial to paid?
- [ ] Average chat credits spent from free trials
- [x] Average chat credits spent from bought users
- [x] Also cost per user in descending order from highest to lowest
- [x] Average number of requests sent per user per month
